subsys/bluetooth/host/Kconfig

```diff
@@ -132,6 +133,7 @@ config BT_RX_STACK_SIZE
 	default 2600 if BT_MESH
 	default 2048 if BT_AUDIO
 	default 2200 if BT_SETTINGS
+	default 1536 if BT_NXP && DEBUG
```

Comment:
why the increase? the hci driver shouldn't have an influence on the host RX stack.

Reply:
Indeed, The increase is not needed. Corrected